packages/ag-charts-community/src/options/chart/chartOptions.ts

```diff
@@ -193,4 +193,6 @@ export interface AgBaseChartOptions<TDatum = any> extends AgBaseThemeableChartOp
      * __Important:__ Make sure to read the `autoSize` config description for information on how the container element affects the chart size (by default).
      */
     container?: HTMLElement | null;
+    /** HTML tabIndex to apply to chart canvas to allow keyboard navigation. Defaults to 0. */
+    tabIndex?: number;
```

Comment:
I'm unsure whether we want to expose this in the options contract just yet. `keyboard.enabled` is undocumented but defaults to `false`, so this option could be confusing because by default it will set `tabindex: -1` and ignore keyboard events.

Reply:
I've removed the options update, and moved the new (hidden) option to `keyboard.tabIndex` for the moment - this should hopefully make the relationship a little clear, subject to final options naming.